apis/apiextensions/v1alpha1/composition_types.go

```diff
@@ -48,14 +48,14 @@ var (
 
 // CompositionSpec specifies the desired state of the definition.
 type CompositionSpec struct {
-	// From refers to the type that this composition is compatible. The values
-	// for the underlying resources will be fetched from the instances of the
-	// From.
+	// CompositeTypeRef refers to the type that this composition is compatible.
+	// The values for the underlying resources will be fetched from the
+	// instances of the referenced composite type.
 	// +immutable
-	From TypeReference `json:"from"`
+	CompositeTypeRef TypeReference `json:"compositeTypeRef"`
 
-	// To is the list of target resources that make up the composition.
-	To []ComposedTemplate `json:"to"`
+	// Resources is the list of target resources that make up the composition.
```

Comment:
```suggestion
	// Resources is the list of resource templates that will be used when a composite resource referring to this composition is created.
```

I think somehow we should say these are resource _templates_